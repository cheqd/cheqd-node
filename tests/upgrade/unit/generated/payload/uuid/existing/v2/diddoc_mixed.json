{
    "did_doc": {
        "context": [
            "abc",
            "def"
        ],
        "id": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
        "controller": [
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d"
        ],
        "verification_method": [
            {
                "id": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-1",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
                "verification_material": "z6Mkhu3KZFd39BSD8kDNbiWyXsXXZ9uy8qMNrqbrkkV9z4YD"
            },
            {
                "id": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-2",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
                "verification_material": "z6MkwUMt89ooTRMGDrPXX1Uei93ouEhuNpgcXtRaazE33Dcw"
            },
            {
                "id": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-3",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
                "verification_material": "z6MksenQTTP7bs4HxfAisuJbh6qxW9ffucX5p5tpPkeQ4kmm"
            },
            {
                "id": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-4",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d",
                "verification_material": "z6MkvLgz1avSwuTpMn9KwHR9edSrkJWypozqJgcLkRc3Ahn5"
            }
        ],
        "authentication": [
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-1",
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-2"
        ],
        "assertion_method": [
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-3"
        ],
        "capability_invocation": [
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-4",
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-1"
        ],
        "capability_delegation": [
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-4",
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-2"
        ],
        "key_agreement": [
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-1",
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-2",
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-3",
            "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#key-4"
        ],
        "service": [
            {
                "id": "did:cheqd:testnet:C1594E93-dEd6-4244-9CC4-74af38dab09d#service-1",
                "serviceType": "type-1",
                "service_endpoint": [
                    "endpoint-1"
                ]
            }
        ],
        "also_known_as": [
            "alias-1",
            "alias-2"
        ]
    },
    "metadata": {
        "created": "2021-01-01T00:00:00Z",
        "updated": "2021-01-01T00:00:00Z",
        "deactivated": false,
        "version_id": "80e5fa13-d60f-49fc-97b3-f75201cd7dde",
        "next_version_id": "",
        "previous_version_id": ""
    }
}