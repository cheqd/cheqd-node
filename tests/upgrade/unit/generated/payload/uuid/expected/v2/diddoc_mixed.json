{
    "did_doc": {
        "context": [
            "abc",
            "def"
        ],
        "id": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
        "controller": [
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884"
        ],
        "verification_method": [
            {
                "id": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-1",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
                "verification_material": "{\"publicKeyMultibase\":\"z4SnGy1Nbodwk2FNfv9Z8gmyXjae7ix72ApgvvUX94qkq\"}"
            },
            {
                "id": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-2",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
                "verification_material": "{\"publicKeyMultibase\":\"zJ26qXuZN7sro7MYpqSWos3Vp5fS3xwSFqsWekiG27zqZ\"}"
            },
            {
                "id": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-3",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
                "verification_material": "{\"publicKeyMultibase\":\"zECXMsD8gGKZprAL2CLLkr1HxgaPpVjGj84ytZUgP9XzP\"}"
            },
            {
                "id": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-4",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884",
                "verification_material": "{\"publicKeyMultibase\":\"zGtRwRLg1cMyMFHJdFiTJoXtrvjF8QvkUcfhQv9e2FUzh\"}"
            }
        ],
        "authentication": [
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-1",
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-2"
        ],
        "assertion_method": [
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-3"
        ],
        "capability_invocation": [
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-4",
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-1"
        ],
        "capability_delegation": [
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-4",
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-2"
        ],
        "key_agreement": [
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-1",
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-2",
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-3",
            "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#key-4"
        ],
        "service": [
            {
                "id": "did:cheqd:testnet:a35d7486-1749-55bf-96d4-683958a9a884#service-1",
                "serviceType": "type-1",
                "service_endpoint": [
                    "endpoint-1"
                ]
            }
        ],
        "also_known_as": [
            "alias-1",
            "alias-2"
        ]
    },
    "metadata": {
        "created": "2021-01-01T00:00:00Z",
        "updated": "2021-01-01T00:00:00Z",
        "deactivated": false,
        "version_id": "80e5fa13-d60f-49fc-97b3-f75201cd7dde",
        "next_version_id": "",
        "previous_version_id": ""
    }
}